lib/cc/engine/analyzers/violation.rb

```diff
@@ -57,7 +61,15 @@ def name
         end
 
         def calculate_points
-          base_points * issue.mass
+          if issue.mass > threshold
+            base_points + overage * points_per
+          else
```

Comment:
This seems unnecessary (`DEFAULT_POINTS` seems unnecessary in general). If the issue mass is not greater or equal than the threshold, it's not an issue at all, and doesn't need points. Is this maybe vestigial from how classic decided if something was worth reporting?